SPECS/lua/lua.spec

```diff
@@ -75,6 +93,11 @@ mv src/luaconf.h src/luaconf.h.template.in
 %patch2 -p1 -z .configure-linux
 %patch3 -p1 -z .configure-compat-all
 %patch4 -p1 -b .CVE-2019-6706
+%patch5 -p1
+%patch6 -p1
+%patch9 -p1
+sed -i '/#define LUA_ROOT/s:/usr/local/:/usr/:' src/luaconf.h
```

Comment:
Is this sed command actually necessary? I cann't find a commit that creates it in `1.0` so its likely part of the the original Lua spec. 

Unless there's a known reason that also applies to `dev`'s version of Lua this should be omitted.

Reply:
I traced this line back to the original import of the spec from Photon, so removing it